tests/containers/rootless_podman.pm

```diff
@@ -76,10 +76,7 @@ sub run {
         validate_script_output "id $testapi::username", sub { /systemd-journal/ };
     }
 
-    if ((is_s390x || is_ppc64le) && check_bsc1192051()) {
-        record_soft_failure("bsc#1192051 - Permission denied for faccessat2");
-        return;
-    }
+    check_bsc1192051();
```

Comment:
maybe better to drop the function ? not many people can say what is bsc#1192051 out of the blue and this function called in single place so you will safe for people who will read this test some time by just moving function code directly here instead of calling function